### PR TITLE
✨ Add page 3 of the certificate

### PR DESCRIPTION
**Pull request by phlgr**
Added the third page of the certificate.
Check out the proof of concept here: https://nf-certifica-page3-pyr2y3vyvjw.herokuapp.com/?id=phlgr

An example page 3 can be found in the Files Changed.
The iPhone frame is not 100% equal, but will be once we get the original from the designer.

Some components are duplicate of one another and will be put in a function later.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
     "typescript": "^4.3.2"
   },
   "dependencies": {
+    "node-fetch": "^2.6.1",
```
Comment (lmachens) [inline package.json:32]: `node-fetch` doesn't include types:
`npm i -D @types/node-fetch`

File: src/api.ts [hunk 2]
```diff
@@ -5,6 +5,16 @@ export type Talent = {
   firstName: string;
   lastName: string;
   courseId: string;
+  capstoneProject: CapstoneProject;
+};
+
+export type CapstoneProject = {
+  desktop: boolean;
```
Comment (lmachens) [inline src/api.ts:12]: The naming could indicate that this is a boolean. wdyt about
```suggestion
  isDesktop: boolean;
```

File: src/certificate.ts [hunk 3]
```diff
@@ -23,6 +24,10 @@ export async function responseCertificate(
 
   renderSecondPage(doc, course.topics);
 
+  doc.addPage();
+
+  await renderThirdPage(doc, talent, talent.capstoneProject);
```
Comment (lmachens) [inline src/certificate.ts:29]: Is it required to have 3 instead of 2 arguments? `capstoneProject` is already part of `talent`.

**Review by lmachens — COMMENTED** [review]
Good job

File: src/api.ts (hunk 2 above)
Comment (phlgr) [reply src/api.ts:12]: Great addition, changed 🤖

**Review by lmachens — APPROVED** [review]
gj

**Pull request merged**